<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiCompanion</class>
 <widget class="QWidget" name="midiCompanionWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>591</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QComboBox" name="midiInputComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>0</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="inputLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>0</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MIDI Input</string>
     </property>
    </widget>
    <widget class="QLabel" name="outputLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MIDI Output</string>
     </property>
    </widget>
    <widget class="QLabel" name="midiMessageDebugLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>MIDI message debug</string>
     </property>
    </widget>
    <widget class="QComboBox" name="midiOutputComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="IPLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mobile HMD IP Address</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>xxx.xxx.xxx.xxx</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="midiConnectionStatus">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection Status</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>780</y>
       <width>185</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>CommandLinkButton</string>
     </property>
    </widget>
   </widget>
 <resources/>
 <connections/>
</ui>
